Haxe/Static/uhx/expose/HxcppRuntime.hx

```diff
@@ -133,7 +133,7 @@ import uhx.internal.Helpers;
   public static function createInlinePodWrapper(size:Int, info:UIntPtr) : VariantPtr {
     var ret = VariantPtr.fromDynamic( InlinePodWrapper.create(size, info) );
 #if debug
-    if (ret.raw & 1 == 1) {
+    if (ret.isExternalPointer()) {
       throw 'Assertion failed: Hxcpp allocated unaligned structure';
```

Comment:
is this (and the other similar checks in this file) right? Looks like either the condition or the assertion message are incorrect...

Reply:
Oh right. I forgot to update the message!